damnit/gui/table.py

```diff
@@ -387,7 +388,7 @@ def comment_item(self, text, comment_id=None):
             item.setData(comment_id, COMMENT_ID_ROLE)
         return item
 
-    def new_item(self, value, column_id, max_diff=0):
+    def new_item(self, value, column_id, max_diff=0, attrs=None):
```

Comment:
Nitpicking, does it even make sense for these arguments to have default values? I think we always want to pass them.

Reply:
You're right, we never call it without them. I think I changed this around a few times while I was working on that refactoring.